log.c

```diff
@@ -0,0 +1,38 @@
+#include <stdarg.h>
+#include <stdbool.h>
+
+#include "log.h"
+
+static bool use_syslog;
+
+void log_lvl(int lvl, const char *fmt, ...)
+{
+	va_list ap;
+
+	va_start(ap, fmt);
+
+	if ( ! use_syslog ) {
+		FILE *out = stderr;
+		switch (lvl) {
+		case LOG_ERR:
+			fputs("Error: ", stderr);
+			break;
+		case LOG_WARNING:
+			fputs("Warning: ", stderr);
+			break;
+		case LOG_DEBUG:
```

Comment:
The current behavior is for `log_info` to log to `stdout`, so I think we should set `out = stdout` as well in `case LOG_INFO`.

Reply:
Indeed, sorry I missed that out. I'll fix.

Reply:
Fixed in updated commit.